src/rpc.rs

```diff
@@ -157,9 +157,9 @@ impl JsonRpcRequestProcessor {
     }
 
     /// Process JSON-RPC request items sent via JSON-RPC.
-    fn get_balance(&self, pubkey: Pubkey) -> Result<(String, i64)> {
+    fn get_balance(&self, pubkey: Pubkey) -> Result<i64> {
```

Comment:
Does that `Result` wrapper add value?

Reply:
Looks like we either wrap here or in the caller, RpcSolImpl.  Seems about even either way